examples/advanced/main.tf

```diff
@@ -280,13 +245,13 @@ module "cos_bucket3" {
   single_site_location                = var.single_site_location
   hard_quota                          = "1000000" #Sets a maximum amount of storage (in bytes) available for a bucket. If it is set to `null` then quota is disabled.
   archive_days                        = null
-  monitoring_crn                      = module.observability_instances.cloud_monitoring_crn
-  activity_tracker_crn                = local.at_crn
   create_cos_instance                 = false
   existing_cos_instance_id            = module.cos_bucket1.cos_instance_id
   kms_encryption_enabled              = false # disable encryption because single site location doesn't support it
   skip_iam_authorization_policy       = true  # Required since cos_bucket1 creates the IAM authorization policy
   retention_enabled                   = false # disable retention for test environments - enable for stage/prod
+  activity_tracker_read_data_events   = false # disable activity_tracker
+  activity_tracker_write_data_events  = false # disable activity_tracker
```

Comment:
There is no need to set these to false - just remove them and allow the default to be used